src/Reflection/ReflectionClass.php

```diff
@@ -694,9 +694,12 @@ public function getDocComment() : string
             return '';
         }
 
-        /* @var \PhpParser\Comment\Doc $comment */
```

Comment:
This change exists in 3 different classes - can we move this to a static utility like `GetFirstDocComment::forNode(NodeAbstract $node)`? It would need to be marked as `@internal`